Library/ViewModels/ProjectPamphletViewModel.swift

```diff
@@ -34,6 +34,12 @@ public protocol ProjectPamphletViewModelOutputs {
   /// Emits a (project, isLoading) tuple used to configure the pledge CTA view
   var configurePledgeCTAView: Signal<(Either<Project, ErrorEnvelope>, Bool), Never> { get }
 
+  var goToDeprecatedManagePledge: Signal<PledgeData, Never> { get }
+
+  var goToDeprecatedViewBacking: Signal<(Project, User?), Never> { get }
```

Comment:
I know we're deprecating stuff but looking at these parameters I wonder if we can improve this by declaring `BackingData` typealias, etc. for `(Project, User?)` tuple

Just a suggestion

Reply:
Good idea